core/kazoo_modb/priv/couchdb/views/services.json

```diff
@@ -10,7 +10,143 @@
     "language": "javascript",
     "views": {
         "audit_logs_by_creation": {
-            "map": "function(doc) { if ( doc.pvt_deleted || doc.pvt_type != 'audit_log') return; emit(doc.pvt_created, {id: doc._id, status: doc.status, reason: doc.reason, message: doc.message, type: doc.audit.changes.type, summary: doc.invoice.summary}); }"
+            "map": [
+                "function(doc) {",
+                "  if ( doc.pvt_deleted || doc.pvt_type != 'audit_log') return;",
+                "  emit(doc.pvt_created, {",
```

Comment:
If two audit logs are created at the same second, this could lead to the lost entries when paginating splits on this timestamp. I would make it a composite key of `[doc.pvt_created, doc._id]` to ensure we can enumerate all audit log docs with identical pvt_created fields.